src/components/Home/HeroBanner/index.tsx

```diff
@@ -0,0 +1,25 @@
+export default function HeroBanner() {
+  // TODO: Make responsive
+  return (
+    <div className="h-[500px] bg-background">
+      <div className="mx-auto grid w-full max-w-[1200px] grid-cols-2 px-[30px]">
+        <div className="flex flex-col items-start justify-center gap-4 text-primary">
+          <h1 className="text-7xl font-black">
+            Tailored raw meals for your pet
+          </h1>
+          <p className="text-lg">Human Grade. Nutrient Dense. Well Balance.</p>
+          <button className="w-[300px] rounded-full bg-primary py-2.5 text-lg text-white">
```

Comment:
you not going to make component for this? can put a todo?

Reply:
added todo